Week1/homework/connection.js

```diff
@@ -0,0 +1,101 @@
+  "INSERT INTO country VALUES ('Afghanistan', 'Asia', 'Southern and Central Asia', 652090, '1919-01-01', 35530081, 45.9, NULL, NULL, NULL, 'Islamic Emirate', NULL, 'Kabul')",
+  "INSERT INTO country VALUES ('Romania', 'Europe', 'Eastern Europe', 238391, '1878-01-01', 19679306, 69.9, NULL, NULL, NULL, 'Republic', NULL, 'Bucharest')",
+  "INSERT INTO country VALUES ('Switzerland', 'Europe', 'Western Europe', 41284, '1499-01-01', 8476005, 79.6, NULL, NULL, NULL, 'Federation', NULL, 'Bern')",
+  "INSERT INTO country VALUES ('Sweden', 'Europe', 'Nordic Countries', 449964, '836-01-01', 9910701, 79.6, NULL, NULL, NULL, 'Constitutional Monarchy', NULL, 'Stockholm')",
+  "INSERT INTO country VALUES ('Syria', 'Asia', 'Middle East', 185180, '1941-01-01', 18269868, 68.5, NULL, NULL, NULL, 'Republic', NULL, 'Damascus')",
+  "INSERT INTO country VALUES ('Tanzania', 'Africa', 'Eastern Africa', 883749, '1961-01-01', 57310019, 52.3, NULL, NULL, NULL, 'Republic', NULL, 'Dodoma')",
+];
+for (let i = 0; i < insert_country.length; i++) {
+  sendQuery(insert_country[i]);
+}
+
+//Create a table called 'city'.
+var create_city =
+  'create table city (ID int, Name varchar(50), Country_Code int, District varchar(50), Population int)';
+
+var insert_city = [
+  "INSERT INTO city VALUES (1, 'Amsterdam', 31, NULL, 821752)",
+  "INSERT INTO city VALUES (2, 'Rotterdam', 31, NULL, 623652)",
+  "INSERT INTO city VALUES (3, 'Kabul', 93, NULL, 4600000)",
+  "INSERT INTO city VALUES (4, 'Bucharest', 40, NULL, 1836000)",
+  "INSERT INTO city VALUES (5, 'Stockholm', 46, NULL, 965232)",
+  "INSERT INTO city VALUES (6, 'Damascus', 963, NULL, 1711000)",
+  "INSERT INTO city VALUES (7, 'Dodoma', 255, NULL, 2084000)",
+  "INSERT INTO city VALUES (8, 'Bern', 41, NULL, 133115)",
+];
+
+sendQuery(create_city);
+for (let i = 0; i < insert_country.length; i++) {
+  sendQuery(insert_city[i]);
+}
+//Write queries (that's a technical term meaning 'requests for information from a database') that will retrieve data that answers the following questions
+//Write queries for: What are the names of the countries with population greater than 8 million
+var select_by_population = 'SELECT name FROM country WHERE population > 8000000';
+sendQuery(select_by_population);
+
+//Write queries for: What are the names of the countries that have “land” in their names ?
+var select_by_land = "SELECT name FROM country WHERE name LIKE '%land%'";
+sendQuery(select_by_land);
+
+//Write queries for: What are the names of the cities with population in between 500,000 and 1 million ?
+var select_query = 'SELECT * FROM city WHERE Population BETWEEN 500000 AND 1000000';
+sendQuery(select_query);
+
+//Write queries for: What are the names of all the countries on the continent ‘Europe’ ?
+var select_by_continent = "SELECT name , continent FROM country WHERE continent = 'Europe'";
```

Comment:
do we really need to have continent in the result. I mean it will be Europe for every row right? Or will it 🤔 ?

Reply:
You're right will remove it 😄